README.md

```diff
@@ -27,10 +27,15 @@ This pack provides three commands: `CommentIssue`, `IssueInfo` and `CreateIssue`
 ### issueInfo command
 This command returns information about a specific issue.
 #### Input
-This commands input is the id of the desired issue:
+This commands input is the id of the desired issue. The issue can either be a URL with an issueId base or an issueId in string representation.
```

Comment:
Suggestion: change to "This command's input" with the apostrophe. Wouldn't be worth changing if you weren't already changing that line.

Reply:
Great catch!